packages/grid_client/src/primitives/nodes.ts

```diff
@@ -47,6 +47,7 @@ interface NodeInfo {
   updatedAt: string;
   total_resources: NodeResources;
   used_resources: NodeResources;
+  farm_ids: number[];
```

Comment:
do we have this on the grid proxy nodes endpoint?

Reply:
you're right, it's not existing